page_object_ecomap/tests/TestRegistration.py

```diff
@@ -22,24 +23,21 @@ def test1_registration(self):
         self.assertTrue(self.reg_page.is_element_present(*RegisterPageLocator.CONFIRMPASSWORD))
         self.assertTrue(self.reg_page.is_element_present(*RegisterPageLocator.SUBMIT_BUTTON))
 
-        enter_value = self.reg_page.reg(self.test_data.get("registration_email") % self.generate_random_email(), self.test_data.get("registration_name"),
-                                     self.test_data.get("registration_surname"), self.test_data.get("registration_nickname") % self.generate_random_email(),
-                                     self.test_data.get("registration_password"),self.test_data.get("registration_confirmpassword"))
-
-    def test2_check_correct_url(self):
-        self.assertEqual(self.home_page.get_current_url(), self.reg_page.get_expected_reg_url())
-
-
-    def test3_check_user_button(self):
+    def test_2_check_registered_user_page(self):
+        self.reg_page.reg(self.test_data.get("registration_email") % self.generate_random_email(),
+                          self.test_data.get("registration_name"),
+                          self.test_data.get("registration_surname"),
+                          self.test_data.get("registration_nickname") % self.generate_random_email(),
+                          self.test_data.get("registration_password"),
+                          self.test_data.get("registration_confirmpassword"))
         self.reg_page.wait_linked_text_changed()
         user_name = self.test_data.get("registration_name") + " " + self.test_data.get("registration_surname")
         att = self.user_page.user_credentials_btn_is_present()
         self.assertEqual(user_name.upper(), att)
-        self.assertTrue(self.user_page.is_element_present(*HomeUserPageLocator.LOGOUT_LINK))
+        self.user_page.is_logout_btn_present()
```

Comment:
this method returns "true", you forgot assertTrue :) missed this at first glance